Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -151,18 +165,18 @@ enum EOSState
         public partial class EOSSingleton
         {
             static private EpicAccountId s_localUserId;
-            static private ProductUserId s_localProductUserId = null;
+            static private ProductUserId s_localProductUserId;
 
             static private NotifyEventHandle s_notifyLoginStatusChangedCallbackHandle;
             static private NotifyEventHandle s_notifyConnectLoginStatusChangedCallbackHandle;
             static private NotifyEventHandle s_notifyConnectAuthExpirationCallbackHandle;
             static private EOSConfig loadedEOSConfig;
 
             // Setting it twice will cause an exception
-            static bool hasSetLoggingCallback = false;
+            static bool hasSetLoggingCallback;
```

Comment:
These kind of changes are an artifact of the tool I used to shorten namespaces - it also removed redundant code. In C#, initializing a `static` variable to something other than it's default value is not necessary.